src/transform/transform.ts

```diff
@@ -72,6 +73,7 @@ function schemaTypeToGoType(schema: Schema): string {
         return 'float32';
       }
       return 'float64';
+    case SchemaType.Any:
```

Comment:
I'm thinking that `Any` should probably be a `interface{}` type.  Thoughts?

Reply:
Ah yes! That's much better than string. I agree. I'll update it.

Reply:
We can start with that and always change it later as needed.